lib/tasks/db.rake

```diff
@@ -1,10 +1,49 @@
 namespace :db do
+  def db_allow_task?
+    if Rails.env.test? || Rails.env.development?
+      true
+    else
+      puts "Only allowed in development and test mode"
+      false
+    end
+  end
 
   desc "Drop database for current RAILS_ENV"
   task :oracle_drop => :environment do
+    next unless db_allow_task?
     config= Rails.configuration.database_configuration[Rails.env]
     connect_string = "#{config["username"]}/#{config["password"]}@#{config["database"]}"
     system "bash -ic 'cd #{Rails.root}/db && ./generate_drops.sh | sqlplus #{connect_string}'"
```

Comment:
I realize this code has been here a long time but that `cd` would fail if there were any spaces in `Rails.root`, and I could see `connect_string` breaking it, depending on the password.